tests/test_portchannel_global.rb

```diff
@@ -45,159 +45,170 @@ def n3k_in_n3k_mode?
-      @global.default_hash_distribution
-    assert_equal(@global.default_hash_distribution,
-                 @global.hash_distribution)
+    global = create_portchannel_global
+    if validate_property_excluded?('portchannel_global', 'hash_distribution')
+      assert_raises(Cisco::UnsupportedError) do
+        global.hash_distribution = 'fixed'
+      end
+      assert_nil(global.hash_distribution)
+    else
+      global.hash_distribution = 'fixed'
+      assert_equal('fixed', global.hash_distribution)
+      global.hash_distribution =
+        global.default_hash_distribution
+      assert_equal(global.default_hash_distribution,
+                   global.hash_distribution)
+    end
   end
 
   def test_get_set_load_defer
-    skip('Platform does not support this property') if n6k_platform? ||
-                                                       n9k_platform?
-    @global = create_portchannel_global
-    @global.load_defer = 1000
-    assert_equal(1000, @global.load_defer)
-    @global.load_defer =
-      @global.default_load_defer
-    assert_equal(@global.default_load_defer,
-                 @global.load_defer)
+    global = create_portchannel_global
+    if validate_property_excluded?('portchannel_global', 'load_defer')
+      assert_raises(Cisco::UnsupportedError) do
+        global.load_defer = 1000
+      end
+      assert_nil(global.load_defer)
+    else
+      global.load_defer = 1000
+      assert_equal(1000, global.load_defer)
+      global.load_defer =
+        global.default_load_defer
+      assert_equal(global.default_load_defer,
+                   global.load_defer)
+    end
   end
 
   def test_get_set_resilient
-    skip('Platform does not support this property') if n6k_platform? ||
-                                                       n7k_platform?
-    @global = create_portchannel_global
-    @global.resilient = true
-    assert_equal(true, @global.resilient)
-    @global.resilient = @global.default_resilient
-    assert_equal(@global.default_resilient, @global.resilient)
+    global = create_portchannel_global
+    if validate_property_excluded?('portchannel_global', 'resilient')
+      assert_raises(Cisco::UnsupportedError) do
+        global.resilient = true
+      end
+      assert_nil(global.resilient)
+    else
+      global = create_portchannel_global
+      global.resilient = true
+      assert_equal(true, global.resilient)
+      global.resilient = global.default_resilient
+      assert_equal(global.default_resilient, global.resilient)
+    end
   end
 
   def test_get_set_port_channel_load_balance_sym_concat_rot
-    skip('Platform does not support this property') if
-      n6k_platform? || n7k_platform? || n3k_in_n3k_mode?
-    @global = create_portchannel_global
-    @global.send(:port_channel_load_balance=,
-                 'src-dst', 'ip-l4port', nil, nil, true, true, 4)
+    # rubocop:disable Style/MultilineOperationIndentation
+    skip('Test not supported on this platform') if n3k_in_n3k_mode? ||
```

Comment:
**Note:** The plan is to replace these skips with the correct unsupported tests, but these tests are going to need a bit of redesign first so this is beyond the scope of this PR.